src/main/resources/assets/appliedenergistics2/lang/en_US.lang

```diff
@@ -345,6 +347,15 @@ gui.tooltips.appliedenergistics2.DoublePattern=Double pattern
 gui.tooltips.appliedenergistics2.DoublePatternHint=Double stack sizes of pattern inputs and output(s)
 gui.tooltips.appliedenergistics2.CraftingStatus=Crafting Status
 gui.tooltips.appliedenergistics2.CraftingStatusDesc=Opens crafting status window
+gui.tooltips.appliedenergistics2.MolecularAssemblerOn=Show Crafting interfaces only
+gui.tooltips.appliedenergistics2.MolecularAssemblerOff=Show all interface types
+gui.tooltips.appliedenergistics2.MolecularAssemblerDesc=Shortcut that toggles interfaces on molecular assemblers
```

Comment:
Keep consistency with `ToggleShowFullInterfaces`. Players will be confused whether "click to show" or "showing".